CHANGELOG.md

```diff
@@ -4,6 +4,56 @@
 
 [1]: https://www.npmjs.com/package/nodejs-paginator?activeTab=versions
 
+## v0.2.0
```

Comment:
What's our policy for pre-major releases, i.e., is there a policy for when we'd move to `1.x.x`. I've tended to just immediately release at `1.x.x`, which simplifies the conventional commit workflow a bit (I've always adhered to the philosophy versions are cheap).

Reply:
Yup.  This is something we've talked about a lot internally, and we're ready to start doing 1.0 releases *after* node 6 EOL.

Reply:
Modules like this are used across a number of our client libraries and serve more as an implementation detail, so I think we can make this 1.x whenever we want. I've always looked to @JustinBeckwith to make that decision.

However for client libraries Googlers tend to be pretty reserved when it comes to making major bumps. Usually 1.x means the library has gone GA and IIRC the request is to try and limit the number of major bumps to a few per year.